gui/main.cpp

```diff
@@ -37,39 +38,24 @@ int main(int argc, char *argv[]) {
   auto startup_time = state_container.GetStartupTime();
 
   std::unique_ptr<ConnectedDevice> device =
-      std::make_unique<FakeConnectedDevice>(
-          [&](const GuiStatus &gui_status) {
-            // For now, completely ignore gui_status.
-            (void)gui_status;
-          },
-          [&](ControllerStatus *controller_status) {
-            // Fill the status with fake data.
-            controller_status->uptime_ms =
-                TimeAMinusB(SteadyClock::now(), startup_time).count();
-            auto *sensors = &controller_status->sensor_readings;
-            sensors->pressure_cm_h2o =
-                sin(controller_status->uptime_ms * 0.001);
-            sensors->volume_ml = 0;
-            sensors->flow_ml_per_min = 0;
-          });
+      std::make_unique<RespiraConnectedDevice>("/dev/pts/6", 10,
```

Comment:
Please keep the old FakeConnectedDevice here by default (so that a contributor who launches the GUI without the scripts or a real device at least sees some fake data), and put the /dev/pts code commented-out instead, and add a TODO to make it configurable using a command-line flag.

Reply:
Done